python/BioSimSpace/_SireWrappers/_system.py

```diff
@@ -53,6 +53,224 @@
+        "GLUH",
+        "HIS1",
+        "HISD",
+        "HISE",
+        "HISH",
+        "LYSH",
+        # from Gromacs 4.5.3 gromos53a6.ff/aminoacids.rtp
+        "ASN1",
+        "CYS1",
+        "HISA",
+        "HISB",
+        "HIS2",
+        # from Gromacs 4.5.3 amber03.ff/aminoacids.rtp
+        "HID",
+        "HIE",
+        "HIP",
+        "ORN",
+        "DAB",
+        "LYN",
+        "HYP",
+        "CYM",
+        "CYX",
+        "ASH",
+        "GLH",
+        "ACE",
+        "NME",
+        # from Gromacs 2016.3 amber99sb-star-ildn.ff/aminoacids.rtp
+        "NALA",
+        "NGLY",
+        "NSER",
+        "NTHR",
+        "NLEU",
+        "NILE",
+        "NVAL",
+        "NASN",
+        "NGLN",
+        "NARG",
+        "NHID",
+        "NHIE",
+        "NHIP",
+        "NTRP",
+        "NPHE",
+        "NTYR",
+        "NGLU",
+        "NASP",
+        "NLYS",
+        "NPRO",
+        "NCYS",
+        "NCYX",
+        "NMET",
+        "CALA",
+        "CGLY",
+        "CSER",
+        "CTHR",
+        "CLEU",
+        "CILE",
+        "CVAL",
+        "CASF",
+        "CASN",
+        "CGLN",
+        "CARG",
+        "CHID",
+        "CHIE",
+        "CHIP",
+        "CTRP",
+        "CPHE",
+        "CTYR",
+        "CGLU",
+        "CASP",
+        "CLYS",
+        "CPRO",
+        "CCYS",
+        "CCYX",
+        "CMET",
+        "CME",
+        "ASF",
+    }
+
+    # A set of nucleic acid residues. Taken from MDAnalysis.
+    nucl_res = {
+        "ADE",
+        "URA",
+        "CYT",
+        "GUA",
+        "THY",
+        "DA",
+        "DC",
+        "DG",
+        "DT",
+        "RA",
+        "RU",
+        "RG",
+        "RC",
+        "A",
+        "T",
+        "U",
+        "C",
+        "G",
+        "DA5",
+        "DC5",
+        "DG5",
+        "DT5",
+        "DA3",
+        "DC3",
+        "DG3",
+        "DT3",
+        "RA5",
+        "RU5",
+        "RG5",
+        "RC5",
+        "RA3",
+        "RU3",
+        "RG3",
+        "RC3",
+    }
+
+    # A list of ion elements.
+    ions = [
+        "F",
+        "Cl",
+        "Br",
+        "I",
+        "Li",
+        "Na",
+        "K",
+        "Rb",
+        "Cs",
+        "Mg",
+        "Tl",
+        "Cu",
+        "Ag",
+        "Be",
+        "Cu",
+        "Ni",
+        "Pt",
+        "Zn",
+        "Co",
+        "Pd",
+        "Ag",
+        "Cr",
+        "Fe",
+        "Mg",
+        "V",
+        "Mn",
+        "Hg",
+        "Cd",
+        "Yb",
+        "Ca",
+        "Sn",
+        "Pb",
+        "Eu",
+        "Sr",
+        "Sm",
+        "Ba",
+        "Ra",
+        "Al",
+        "Fe",
+        "Cr",
+        "In",
+        "Tl",
+        "Y",
+        "La",
+        "Ce",
+        "Pr",
+        "Nd",
+        "Sm",
+        "Eu",
+        "Gd",
+        "Tb",
+        "Dy",
+        "Er",
+        "Tm",
+        "Lu",
+        "Hf",
+        "Zr",
+        "Ce",
+        "U",
+        "Pu",
+        "Th",
+    ]
```

Comment:
Could all of these be moved _outside_ of the class? I'd rather they were hidden from the user since they are only used internally. It would probably be best to place them in a utility module, e.g. `_utils.py`, then import them privately in `_system.py` to use where required, e.g.:

```python
from ._utils import prot_res as _prot_res
...
```

For simplicity, they could even be private in the `_utils.py` module itself, e.g. just call it `_prot_res`, etc.

Reply:
Of course - have updated and will push shortly.